Capstone/All Comments/DetailCommentCell.h

```diff
@@ -0,0 +1,30 @@
+//
+//  DetailCommentCell.h
+//  Capstone
+//
+//  Created by Abel Kelbessa on 8/8/22.
+//
+
+#import <UIKit/UIKit.h>
+#import "Comment.h"
+#import <Parse/Parse.h>
+#import "Parse/PFImageView.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DetailCommentCell : UITableViewCell
+@property (weak, nonatomic) IBOutlet UILabel *usernameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *userCommentLabel;
+@property (weak, nonatomic) IBOutlet UIButton *upvoteHeartButton;
+@property (weak, nonatomic) IBOutlet UIButton *downvoteHeartButton;
+@property (weak, nonatomic) IBOutlet UILabel *counterLabel;
+@property (weak, nonatomic) IBOutlet PFImageView *userProfileImageView;
+@property (weak, nonatomic) NSString *commentObjectId;
+@property (weak, nonatomic) Comment *comment;
```

Comment:
<img width="952" alt="Screen Shot 2022-08-10 at 8 11 19 AM" src="https://user-images.githubusercontent.com/96321082/183940545-399891bf-f581-4079-bf71-2c9275ad9018.png">
I believe that it is needed to pass the comment from Comments View Controller. If the public comment variable is removed, there would not be any possible way to pass the original comment from the comments view controller and update the vote count in Detail Comment Cell. Is there a better way to do this that I am not seeing?

Reply:
mm i see, that makes sense! if you're directly passing in `comment` via the CommentsViewController, then do we need to query at all? we could easily use the original `comment` to make our updates, etc. instead of querying for a new `commentFromParse`

and then that way, instead of having a `commentFromParse` variable at all, we just use `comment`

Reply:
I originally attempted to use the original comment to update its votes. However, that did not work when I did [self.comment saveInBackground]; to update the object's vote in parse after incrementing locally. That is the reason that I queried for the comment using commentFromParse.